### ISSUE TITLE
[Bug]: Prometheus scrape endpoint responds with invalid Content-Type

### ISSUE THREAD

**Issue by Luuk-Horsman-ISNL** [strimzi/strimzi-kafka-operator#10985]
### Bug Description

With the new version of Prometheus 3. Custom scrape config resources like Probe, ServiceMonitor, etc. Do not assume the default Content-Type PrometheusText1.0.0 anymore. This is a [breaking change](https://prometheus.io/docs/prometheus/latest/migration/#scrape-protocols) on the Prometheus side. Therefore the metrics endpoints should respond with the correct Content-Type header.

Right now the Content-Type header is not PrometheusText1.0.0 or any of the other formats that Prometheus supports. Which causes errors and causes Prometheus to not ingest metrics anymore. 

There is a workaround, but it would be more elegant to resolve it in the source project. The workaround would be to add `fallbackScrapeProtocol: PrometheusText1.0.0` to the spec in the Prometheus resources.

### Steps to reproduce

1. Have strimzi-kafka-operator installed in cluster.
2. Have Prometheus operator with Prometheus 3 installed in cluster.
3. Create a podmonitor that looks like the following:
```yaml
apiVersion: monitoring.coreos.com/v1
kind: PodMonitor
metadata:
  name: cluster-operator-metrics
  labels:
    app: strimzi
spec:
  selector:
    matchLabels:
      strimzi.io/kind: cluster-operator
  namespaceSelector:
    matchNames:
      - '{{ .Release.namespace }}'
  podMetricsEndpoints:
  - path: /metrics
    port: http

```
4. Check the prometheus logs. Error should be throwing because of the wrong Content-Type.

Like i said, setting the `fallbackScrapeProtocol: PrometheusText1.0.0` property in the spec should fix the error. But the /metrics endpoint should respond with the correct Content-Type aswell.

### Expected behavior

The /metrics endpoint should return a proper Content-Type header like PrometheusText1.0.0.

### Strimzi version

0.44.0

### Kubernetes version

1.30

### Installation method

Helm chart

### Infrastructure

AKS

**Comment by scholzj** [comment]
This was addressed in #10908 / 0.45.0 release.

**Comment by Luuk-Horsman-ISNL** [comment]
Alright, thanks for the reply.
